draft-ietf-tls-rfc8446bis.md

```diff
@@ -5544,7 +5544,11 @@ Forward secret with respect to long-term keys:
   the handshake is complete, this does not compromise the security of the
   session key (see {{?DOW92=DOI.10.1007/BF00124891}}), as long as the session key itself has
   been erased. The forward secrecy property is not satisfied
-  when PSK is used in the "psk_ke" PskKeyExchangeMode.
+  when PSK is used in the "psk_ke" PskKeyExchangeMode. If key shares, shared secrets,
+  or any keys derived in the TLS Key Schedule except binder_key, resumption_secret,
+  and PSKs derived from the resumption_secret are not erased short-term after
+  closure of a connection this creates additional long-term keys, and the property
+  is no longer satisfied.
```

Comment:
```suggestion
  when PSK is used in the "psk_ke" PskKeyExchangeMode.  Failing to erase keys or secrets intended to be ephemeral or connection-specific in effect creates additional long-term keys that must be protected, and compromise of those long-term keys (even after the handshake is complete) can result in loss of secrecy of the connection's traffic.
```

Reply:
Avoid the comma splice on "protected, and compromise" -> "protected. Compromise"

Reply:
I fixed the comma splice